packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -379,6 +377,7 @@ export class StatsAnalyzer extends EventsScope {
     });
 
     newMqa.intervalMetadata.peerReflexiveIP = this.statsResults.connectionType.local.ipAddress;
+    newMqa.intervalMetadata.cpuInfo.numberOfCores = CpuInfo.getNumLogicalCores();
```

Comment:
We're unlikely to run into this since all the major browsers support this feature, but what happens when `CpuInfo.getNumLogicalCores()` is undefined? I would assume it would fail validation.

I think there was a conversation in the "Ask MQA" space about, if we are unable to get any information about the CPU, we should just leave `newMqa.intervalMetadata.cpuInfo` undefined. Do you think we should do this here?

Reply:
Thank you for pointing this out. I believe it's worth handling it as `undefined` too.